server/src/server.js

```diff
@@ -1,13 +1,20 @@
 import express from 'express'
 import cors from 'cors'
+import { connect } from './db'
+import { restRouter } from './api'
 
 const app = express()
+connect()
```

Comment:
Just a question here, do we need body-parser middleware?

Reply:
Good question, I had it originally but removed it because I thought I read somewhere that it's part of Express now. Didn't have time to research it, can definitely add to the next PR after I double check

Reply:
It is part of Express now but you have to use the middleware:

```js
app.use(express.json())
```